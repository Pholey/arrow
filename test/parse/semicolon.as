// Semicolons are treated as the null statement.
;;;;;;  ;;;;;;  ;;;;;;  ;;
;;      ;;  ;;  ;;  ;;  ;;
;;;;;;  ;;;;;;  ;;;;;;  ;;;;;;
