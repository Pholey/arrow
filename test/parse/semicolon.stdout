{
  "tag": "Module",
  "span": "test/parse/semicolon.as:1:1",
  "statements": []
}
